lib/config.js

```diff
@@ -323,6 +323,16 @@ const calculateValue = () => {
         zodgame: {
             cookie: envs.ZODGAME_COOKIE,
         },
+        openai: {
```

Comment:
Since it's not a route specific config, please put it above
https://github.com/DIYgod/RSSHub/blob/d1cb63d62adf95ce479ec427639af44465a5a022/lib/config.js#L116